public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -161,42 +152,68 @@ export function RichHistoryCard(props: Props) {
   } = props;
   const [activeUpdateComment, setActiveUpdateComment] = useState(false);
   const [comment, setComment] = useState<string | undefined>(query.comment);
-  const [queryDsInstance, setQueryDsInstance] = useState<DataSourceApi | undefined>(undefined);
+  const { value, loading } = useAsync(async () => {
```

Comment:
This is the nittyest of nits, but what do you think about something besides `value` for this variable name? I know it's literally the value of the rich history card so your variable name makes sense, but we have areas reading `queries` values from props and some from this, it might be helpful to either define this variable name or leave a comment. Maybe like `dsPopulatedData` `dsEnhancedRecord` for possible variable names?

Reply:
tbh i think with a few rafactorings that happened variable names in this component went all bonkers (like `queryDsInstance`, `dsInstance`, `query` (which is instead `historyEntry`) ), i actually settled for `value` because it felt the lcearer to me :D

i wanted to rename ~all the things but didn't want to pollute the PR with a bunch of renaming that would have made reviewing harder. if it's ok tho i'll just push an extra commit to do all the renaming, especially. wdyt?

Reply:
Nah, I understand where you're coming from. Feel free to leave it, it's easy enough to understand what's going on

Reply:
i'll create another issue when merging this and a follow up pr to address all the ambiguous names in this file